### DISCUSSION TITLE
Intercept scheduled requests

### DISCUSSION THREAD

**Discussion by paichinger** [metalbear-co/mirrord#1999] [General]
Hey!
I'm using telepresence and I have the following problem:
I intercept a service running in k8s and route all its traffic to the the same application running on my machine. Works!
But the application I'm intercepting has some scheduled tasks built in an does request to other services every once in a while.
Now those get triggered twice: Once from the app running in k8s and once from the one running on my machine. I'd prefer is just the ones from my maching would be triggered.
I think telepresence doesn't support this scenario.
I was wondering if mirrord could do this?
Thanks,
Peter

**Comment by eyalb181** [comment]
Hey Peter!

We have the [pause](https://mirrord.dev/docs/guides/pause/) feature which might support what you're describing. When you run mirrord with pause, it pauses the target pod, so the scheduled tasks you mentioned should only be triggered on your local machine. Any incoming traffic to the target pod will still be routed to your machine.
What do you think, does that answer your requirement?

Thanks,
Eyal
